app/uk/gov/hmrc/helptosave/controllers/HelpToSaveAuth.scala

```diff
@@ -46,10 +46,10 @@ class HelpToSaveAuth(htsAuthConnector: AuthConnector) extends BaseController wit
     Action.async { implicit request ⇒
       authorised(AuthWithCL200)
         .retrieve(Retrievals.nino) { mayBeNino ⇒
-          mayBeNino.fold(
-            toFuture(InternalServerError("could not find NINO for logged in user"))
+          mayBeNino.fold[Future[Result]](
+            Forbidden
```

Comment:
this case never happens ideally , but can we add a log line here saying something like "nino not found for a logged in user" ?

Reply:
yes, I'll add that back in